docs/docs/user/migration-guide/v2-v3.md

```diff
@@ -63,3 +63,21 @@ steps:
       command: deploy --execute
       # ...
 ```
+
+## [Allow overriding config files](https://github.com/bakdata/kpops/pull/391)
+
+Specifying the environment is no longer mandatory. If not defined, only the default files will be used.
+
+`environment` is no longer specified in `config.yaml`. Instead, it can be either set via the cli flag `--environment` or with the environment variable `KPOPS_ENVIRONMENT`.
```

Comment:
```suggestion
`environment` is no longer specified in `config.yaml`. Instead, it can be either set via the CLI flag `--environment` or with the environment variable `KPOPS_ENVIRONMENT`.
```